.github/workflows/build-config-installer.yml

```
@@ -0,0 +1,334 @@
+          # SHA256
+          $sha256 = (Get-FileHash -Path $installerPath -Algorithm SHA256).Hash
+          Write-Host "SHA256: $sha256" -ForegroundColor Green
+          
+          # MD5
+          $md5 = (Get-FileHash -Path $installerPath -Algorithm MD5).Hash
+          Write-Host "MD5: $md5" -ForegroundColor Green
+          
+          # Save to file
+          $checksumFile = "Output\CHECKSUMS.txt"
+          @"
+RustDesk Config Installer Checksums
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)